src/main/java/io/strimzi/kafka/bridge/http/HttpBridge.java

```diff
@@ -194,13 +194,23 @@ private void processRequests(HttpServerRequest httpServerRequest) {
                     this.httpSinkEndpoints.remove(deleteInstanceID);
                 } else {
                     httpServerRequest.response()
-                            .setStatusCode(404)
+                            .setStatusCode(ErrorCodeEnum.UNKNOWN_REQUEST.getValue())
+                            .setStatusMessage("endpoint not found")
                             .end();
                 }
                 break;
-
+            case EMPTY:
+                httpServerRequest.response()
+                        .setStatusCode(ErrorCodeEnum.EMPTY_REQUEST.getValue())
+                        .setStatusMessage("records cannot not be empty")
```

Comment:
we don't know which kind of request this is. Cannot say nothing about the payload and what it should contain. It could be a POST request related to commit offsets so it doesn't contain records.